clap_complete/tests/testsuite/bash.rs

```diff
@@ -99,19 +99,27 @@ fn value_terminator() {
 #[cfg(feature = "unstable-dynamic")]
 #[test]
 fn register_minimal() {
-    use clap_complete::dynamic::Completer;
+    if !common::has_command("bash") {
+        return;
+    }
 
-    let name = "my-app";
-    let bin = name;
-    let completer = name;
-
-    let mut buf = Vec::new();
-    clap_complete::dynamic::shells::Bash
-        .write_registration(name, bin, completer, &mut buf)
-        .unwrap();
-    snapbox::Assert::new()
-        .action_env("SNAPSHOTS")
-        .matches_path("tests/snapshots/register_minimal.bash", buf);
+    common::register_example("test-dynamic", completest::Shell::Bash);
+}
+
+#[test]
+#[cfg(all(unix, feature = "unstable-dynamic"))]
+fn complete_dynamic() {
+    if !common::has_command("bash") {
+        return;
+    }
+
+    let term = completest::Term::new();
+    let mut runtime = common::load_runtime("test-dynamic", completest::Shell::Bash);
+
+    let input = "test-dynamic \t\t";
+    let expected = r#"%"#;
```

Comment:
I don't understand bash completions much, cannot really tell why they are not working.